README.md

```diff
@@ -250,6 +251,38 @@ Inside [`paperqa/configs`](paperqa/configs) we bundle known useful settings:
 | wikicrow     | Setting to emulate the Wikipedia article writing used in our WikiCrow publication.                                           |
 | contracrow   | Setting to find contradictions in papers, your query should be a claim that needs to be flagged as a contradiction (or not). |
 | debug        | Setting useful solely for debugging, but not in any actual application beyond debugging.                                     |
+| tier1_limits | Settings that match OpenAI rate limits for each tier, you can use `tier<1-5>_limits` to specify the tier.                    |
+
+### Rate Limits
+
+If you are hitting rate limits, say with the OpenAI Tier 1 plan, you can add them into PaperQA2.
+For each OpenAI tier, a pre-built setting exists to limit usage.
+
+```bash
+pqa --settings 'tier1_limits' ask 'Are there nm scale features in thermoelectric materials?'
+```
+
+This will limit your system to use the tier1_limits, and slow down your queries to accommodate.
+
+You can also specify them manually:
+
+```bash
+pqa --summary_llm_config '{"rate_limit": {"gpt-4o-2024-08-06": "30000 per 1 minute"}}' ask 'Are there nm scale features in thermoelectric materials?'
+```
+
+Or by adding into a settings object, if calling imperatively:
+
+```python
+from paperqa import Settings, ask
+
+answer = ask(
+    "What manufacturing challenges are unique to bispecific antibodies?",
+    settings=Settings(
+        llm_config={"rate_limit": {"gpt-4o-2024-08-06": "30000 per 1 minute"}},
```

Comment:
Can you document where this `30000 per 1 minute` string comes from? Like can I change the time base to be "per 2 minutes" and would that still work

I don't know what I think of these "free text" settings, it always leaves one wondering if the internal system actually respects or understands it

Reply:
if we use the objects then we can't have serialized settings in JSON, these come from `limits` and i'll link the docs.

Reply:
Okay yeah then link the docs both in source code and `README` if it's cool